gobblin-runtime/src/main/java/gobblin/runtime/TaskStateCollectorService.java

```diff
@@ -160,6 +190,17 @@ public Void call() throws Exception {
       this.jobState.addTaskState(taskState);
     }
 
+    // Finish any addtional steps defined in handler globally.
+    // Currently implemented handler for Hive registration only.
+    if (optionalTaskCollectorHandler.isPresent() &&
+        this.jobState.getPropAsBoolean(ConfigurationKeys.PIPELINED_HIVE_REGISTRATION_ENABLED)){
+      LOGGER.info("Enabled Pipelined HiveRegistrationPublisher for " + taskStateQueue.size() + " tasks");
```

Comment:
The logging here should be removed since is is only for testing. If you need the timings then add them to the metrics.